### DISCUSSION TITLE
the Purge line interfered with the brim

### DISCUSSION THREAD

**Discussion by 54iceblock** [OpenNeptune3D/OpenNept4une#242] [Q&A]
Today I started printing with relative large brim (~10mm) and noticed the purged line is within the brim, then he nuzzle knocked off the pruged line.  Is there a way to move the purged line somewhere, similar to what original Neptune4 pro does?

Thanks!

**Comment by halfmanbear** [comment]
You cann either adjust the;
`variable_purge_margin: 15` within `KAMP_Settings.cfg`.

Or alternatively;

https://github.com/OpenNeptune3D/OpenNept4une/discussions/248#discussion-7815934
